tests/suites/test_suite_ssl.function

```diff
@@ -3014,13 +3062,20 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C:!MBEDTLS_USE_PSA_CRYPTO:MBEDTLS_PKCS1_V15 */
-void handshake( const char *cipher, int version, int pk_alg )
+void handshake( const char *cipher, int version, int pk_alg,
+                const char *psk_str )
 {
     /* forced_ciphersuite needs to last until the end of the handshake */
     int forced_ciphersuite [2];
     enum { BUFFSIZE = 1024 };
     mbedtls_endpoint client, server;
-
+#if defined(MBEDTLS_KEY_EXCHANGE__SOME__PSK_ENABLED)
+    unsigned char psk[MBEDTLS_PSK_MAX_LEN];
```

Comment:
Consider allocating on the heap if this is of any significant size, to help make the test more portable to stack-constrained systems

Reply:
This is usually 32 bytes, so not that much.

Reply:
OK